src/coordinator/users.spec.js

```diff
@@ -850,4 +850,135 @@ describe('Coordinator/Users', function() {
       });
     });
   });
+
+  describe('subscribeEvent', function() {
+    context('when all the required parameters are provided', function() {
+      let expectedUserEvent;
+      let promise;
+      let request;
+      let event;
+      let user;
+      let userEventFromServer;
```

Comment:
If you change the other variable, I'd change this one to `subscriptionFromServer` as well

Reply:
👍